functions/menu.bash

```diff
@@ -2,11 +2,22 @@
 
 show_about() {
   whiptail --title "About openHABian and openhabian-config" --msgbox "openHABian Configuration Tool $(get_git_revision)
-  \\nThis tool provides a few routines to make your openHAB experience as comfortable as possible. The menu options help with the setup and configuration of your system. Please select a menu entry to learn more.
-  \\nVisit the following websites for more information:
+This tool provides a little help to make your openHAB experience as comfortable as possible.
+\\nMake sure you have read the README and know about the Debug and Amanda Backup guides in /opt/openhabian/docs.
```

Comment:
I would not explicitly name Amanda here, just backup would be good (do not confuse newbies)

Reply:
ok